admin/templates/login.html

```diff
@@ -0,0 +1,70 @@
+{% load render_bundle from webpack_loader %}
+{% load staticfiles %}
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>OSFAdmin | Log in</title>
+    <!-- Tell the browser to be responsive to screen width -->
+    <meta content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no' name='viewport'>
+    <!-- Bootstrap 3.3.4 -->
+    <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet" type="text/css" />
```

Comment:
I think @zkraime is fixing these static references but they don't work. Referring back to `base.html`, these should point to `vendor/bower/something`, not sure how well that will work once we're in prod and nginx is serving static files. This page is also missing jQuery right now and throws an error regarding that.